packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -130,6 +135,10 @@ RocketChat.saveUser = function(userId, userData) {
 			updateUser.$set.name = userData.name;
 		}
 
+		if (userData.role) {
+			updateUser.$set.roles = [ (userData.role || 'user') ];
+		}
+
```

Comment:
@rodrigok @engelgabriel True, overriding the values will be simpler and straightforward. Don't you think the roles values should be an array instead of string?

@rodrigok Do you mean to use methods to add/remove roles rather that database query?